### PR TITLE
fix: remove ALL in time periods (temporary until  v2 backend data available)

### PR DESCRIPTION
**Pull request by lynnshaoyu**
https://uniswaplabs.atlassian.net/browse/WEB-1404?atlOrigin=eyJpIjoiOTEwMjM1MmQwNTc0NDMxZmE3NzkxYzIzOGQ2MDc3YzMiLCJwIjoiaiJ9

after change:
Screen Shot 2022-10-03 at 12 18 05 PM
Screen Shot 2022-10-03 at 12 18 11 PM

### CODE REVIEW

File: src/components/Tokens/TokenDetails/PriceChart.tsx [hunk 1]
```diff
@@ -276,8 +276,8 @@ export function PriceChart({ width, height, prices }: PriceChartProps) {
   const crosshairEdgeMax = width * 0.85
   const crosshairAtEdge = !!crosshair && crosshair > crosshairEdgeMax
 
-  /* Default curve doesn't look good for the HOUR/ALL chart */
-  const curveTension = timePeriod === TimePeriod.ALL ? 0.75 : timePeriod === TimePeriod.HOUR ? 1 : 0.9
+  /* Default curve doesn't look good for the HOUR chart */
```
Comment (zzmp) [inline src/components/Tokens/TokenDetails/PriceChart.tsx:279]: Can you make this comment more informative? Do you know _why_ it doesn't look good?
Including that could help future maintainers who might add more TimePeriod choices.

File: src/graphql/data/util.ts [hunk 2]
```diff
@@ -12,6 +12,8 @@ export enum TimePeriod {
   ALL,
 }
 
+export type TimePeriodWithoutAll = Exclude<TimePeriod, TimePeriod.ALL>
```
Comment (zzmp) [inline src/graphql/data/util.ts:15]: Why not just remove ALL from TimePeriod for now? That seems less bug-prone than having two nearly-compatible enums be exported.

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/graphql/data/util.ts (hunk 2 above)
Comment (lynnshaoyu) [reply src/graphql/data/util.ts:15]: makes sense done

File: src/components/Tokens/TokenDetails/PriceChart.tsx (hunk 1 above)
Comment (lynnshaoyu) [reply src/components/Tokens/TokenDetails/PriceChart.tsx:279]: sounds good done

File: src/components/Tokens/TokenDetails/PriceChart.tsx [hunk 3]
```diff
@@ -276,8 +259,10 @@ export function PriceChart({ width, height, prices }: PriceChartProps) {
   const crosshairEdgeMax = width * 0.85
   const crosshairAtEdge = !!crosshair && crosshair > crosshairEdgeMax
 
-  /* Default curve doesn't look good for the HOUR/ALL chart */
-  const curveTension = timePeriod === TimePeriod.ALL ? 0.75 : timePeriod === TimePeriod.HOUR ? 1 : 0.9
+  /* Default curve doesn't look good for the HOUR chart.
```
Comment (zzmp) [inline src/components/Tokens/TokenDetails/PriceChart.tsx:262]: Reformat (I'm surprised your linter does not do this):

```
/*
 * Default curve doesn't look good for the HOUR chart.
 * Higher values make the curve more rigid, lower values smooth the curve but make it less "sticky" to real data points,
 * making it unacceptable for longer durations / larger variances.
 */
```

**Review by zzmp — APPROVED** [review]
lgtm but please update the comment (formatting and including the reason it is unacceptable)

File: src/components/Tokens/TokenDetails/PriceChart.tsx (hunk 3 above)
Comment (lynnshaoyu) [reply src/components/Tokens/TokenDetails/PriceChart.tsx:262]: oh weird need to fix my linter then. 

thanks done.

**Pull request merged**